lib/metrics/amplitude.js

```
@@ -0,0 +1,200 @@
+    event: 'blocked'
+  },
+  'account.login.confirmedUnblockCode': {
+    group: GROUPS.login,
+    event: 'unblock_success'
+  },
+  'account.reset': {
+    group: GROUPS.login,
+    event: 'forgot_complete'
+  },
+  'account.signed': {
+    group: GROUPS.activity,
+    event: 'cert_signed'
+  },
+  'account.verified': {
+    group: GROUPS.registration,
+    event: 'email_confirmed'
+  },
+  'flow.complete': {
+    isDynamicGroup: true,
+    group: (request, data, metricsContext) => GROUPS[metricsContext.flowType],
+    event: 'complete'
+  },
+  'password.forgot.resend_code.completed': {
+    group: GROUPS.login,
+    event: 'forgot_sent'
+  },
+  'password.forgot.send_code.completed': {
+    group: GROUPS.login,
+    event: 'forgot_sent'
+  },
+  'sms.installFirefox.sent': {
+    group: GROUPS.sms,
+    event: 'sent'
+  }
+}
+
+const EMAIL_EVENTS = /^email\.(\w+)\.(bounced|sent)$/
+
+const EMAIL_TYPES = {
+  newDeviceLoginEmail: 'login',
+  passwordChangedEmail: 'change_password',
+  passwordResetEmail: 'reset_password',
+  passwordResetRequiredEmail: 'reset_password',
+  postRemoveSecondaryEmail: 'secondary_email',
+  postVerifyEmail: 'registration',
+  postVerifySecondaryEmail: 'secondary_email',
+  recoveryEmail: 'reset_password',
+  unblockCode: 'unblock',
+  verificationReminderFirstEmail: 'registration',
+  verificationReminderSecondEmail: 'registration',
+  verificationReminderEmail: 'registration',
+  verifyEmail: 'registration',
+  verifyLoginEmail: 'login',
+  verifySyncEmail: 'registration',
+  verifySecondaryEmail: 'secondary_email'
+}
+
+const NOP = () => {}
+
+const EVENT_PROPERTIES = {
+  [GROUPS.activity]: NOP,
+  [GROUPS.email]: mapEmailType,
+  [GROUPS.login]: NOP,
+  [GROUPS.registration]: NOP,
+  [GROUPS.sms]: NOP
+}
+
+const USER_PROPERTIES = {
+  [GROUPS.activity]: mapUid,
+  [GROUPS.email]: mapUid,
+  [GROUPS.login]: mapUid,
+  [GROUPS.registration]: mapUid,
+  [GROUPS.sms]: NOP
+}
+
+module.exports = log => {
+  if (log) {
```

I don't expect it to, just did it so things fail faster if it ever gets forgotten.